curriculum/test/utils/sort-challenges.test.js

```diff
@@ -0,0 +1,252 @@
+/* global expect */
+const { sortChallenges } = require('./sort-challenges');
+
+const challenges = [
+  {
+    name: 'HTML - project 1 - step 1',
+    superOrder: 1,
+    order: 1,
+    challengeOrder: 1
+  },
+  {
+    name: 'HTML - project 1 - step 2',
+    superOrder: 1,
+    order: 1,
+    challengeOrder: 2
+  },
+  {
+    name: 'HTML - project 1 - step 3',
+    superOrder: 1,
+    order: 1,
+    challengeOrder: 3
+  },
+  {
+    name: 'HTML - project 1 - step 4',
+    superOrder: 1,
+    order: 1,
+    challengeOrder: 4
+  },
+  {
+    name: 'HTML - project 2 - step 1',
+    superOrder: 1,
+    order: 2,
+    challengeOrder: 1
+  },
+  {
+    name: 'HTML - project 2 - step 2',
+    superOrder: 1,
+    order: 2,
+    challengeOrder: 2
+  },
+  {
+    name: 'HTML - project 2 - step 3',
+    superOrder: 1,
+    order: 2,
+    challengeOrder: 3
+  },
+  {
+    name: 'HTML - project 2 - step 4',
+    superOrder: 1,
+    order: 2,
+    challengeOrder: 4
+  },
+  {
+    name: 'HTML - project 3 - step 3',
```

Comment:
Noticed this small typo (was in my original suggestion):
```suggestion
    name: 'HTML - project 3 - step 1',
```